.github/workflows/pythonpackage.yml

```diff
@@ -19,19 +26,19 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        python -m pip install umap-learn
-        python -m pip install .
+        python -m pip install ".[test]"
         # python -m pip install mofapy2
         python -m pip install git+https://github.com/bioFAM/mofapy2
         python -m pip install git+https://github.com/scverse/mudata
```

Comment:
This should be

```
python -m pip install ".[test]" git+https://github.com/bioFAM/mofapy2 git+https://github.com/scverse/mudata
```

otherwise all but the third line are basically meaningless.